{
  "metadata": {
    "timestamp": "2025-02-27T04:13:47Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2022-2996",
        "description": "A flaw was found in the python-scciclient when making an HTTPS connection to a server where the server's certificate would not be verified. This issue opens up the connection to possible Man-in-the-middle (MITM) attacks.",
        "metadata": {
          "published": "",
          "last_modified": "",
          "confidence_level": "low",
          "severity": "high",
          "source": "",
          "status": ""
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": false
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:N",
            "attack_vector": "NETWORK",
            "base_score": 7.40000009536743
          }
        ],
        "timeline": {
          "references": null,
          "repositories": null
        },
        "epss": {
          "score": 0.00101,
          "percentile": 0.43213
        },
        "counts": {
          "public_exploit_count": 0
        }
      }
    }
  ]
}